flow/Makefile

```diff
@@ -475,7 +475,8 @@ $(if $(5),$(5),$(RESULTS_DIR))/$(1)$(if $(4),$(4),.odb): $(2)
 
 .PHONY: do-$(1)
 do-$(1):
-	($(TIME_CMD) $(OPENROAD_CMD) $(SCRIPTS_DIR)/$(3).tcl -metrics $(LOG_DIR)/$(1).json) 2>&1 | tee $(LOG_DIR)/$(1).log
+	{ $(TIME_CMD) $(OPENROAD_CMD) $(SCRIPTS_DIR)/$(3).tcl -metrics $(LOG_DIR)/$(1).json && cp $(LOG_DIR)/$(1).tmp.log $(LOG_DIR)/$(1).log; } 2>&1 | tee $(LOG_DIR)/$(1).tmp.log
+	rm -f $(LOG_DIR)/$(1).tmp.log
```

Comment:
why cp and not mv?

Reply:
The file still exists.

Reply:
Not sure I follow, I copy and delete, I dont move it.

Reply:
because the file is still open `(a&&b) | tee foo.txt`, here foo.txt is open until b has completed.

However, I'm testing what you expected now. moving an open file is fine in Linux. Will update PR  after some more testing.

Reply:
There is no need to use && - just mv it after it finishes.  If it errors the step should stop before mv is executed.

Reply:
Indeed.

Reply:
Fixed